temporal-serviceclient/src/main/java/io/temporal/serviceclient/DefaultServiceOperationRpcRetryOptions.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.serviceclient;
+
+import io.grpc.Status;
+import io.temporal.api.errordetails.v1.QueryFailedFailure;
+import java.time.Duration;
+
+public class DefaultServiceOperationRpcRetryOptions {
+  public static final Duration RETRY_SERVICE_OPERATION_INITIAL_INTERVAL = Duration.ofMillis(20);
```

Comment:
Initial interval has been recently increased to 200ms in go to avoid spamming the server too much.

Reply:
Thanks for pointing it out! I don't want to put too many meaningful changes in this PR, because it's heavy in refactoring. I will make this change in a follow-up PR.